src/Adapter/MSTestAdapter.PlatformServices/Services/MSTestAdapterSettings.cs

```diff
@@ -46,6 +47,10 @@ public MSTestAdapterSettings()
     /// </summary>
     protected List<RecursiveDirectoryPath> SearchDirectories { get; private set; }
 
+#if !NETFRAMEWORK
```

Comment:
I think (might be wrong) that you could remove the compiler directive everywhere but in the Validate method.

Reply:
gives me runtime errors